developer/index.twig

```diff
@@ -6,9 +6,15 @@
   </head>
   <body>
     {% set masthead_lockup_line1 = "Decanter Preview Environment" %}
+    {% set masthead_lockup_modifier_class = "su-lockup--option-n" %}
     {%- include "@decanter/components/masthead/masthead.twig" -%}
     <main>
-      {%- include "./test.twig" -%}
+      <div class="fullwidth">
+        {%- include "./test.twig" -%}
+      </div>
+      <div class="custom-width" style="width: 700px;">
+        {%- include "./test.twig" -%}
+      </div>
```

Comment:
I'd like to have this index.twig file open for full page mock ups if we can. Feel free to put this in your test.twig file for your own work.